### DISCUSSION TITLE
windows界面不显示

### DISCUSSION THREAD

**Discussion by rock-dengpaoer** [qier222/YesPlayMusic#1148] [Q&A] [answered]
# 尽量每个 issue 只提一个 bug 或新功能

### 提新 issue 前请确认 👉

- 没人提过这个 issue（[这里看所有 issue](https://github.com/qier222/YesPlayMusic/issues)）
- 项目的 Todo 里没有与你 issue 相关的内容（[这里看 Todo](https://github.com/qier222/YesPlayMusic/projects/1)）

### 反馈 bug 需要的信息

- 用的是网页版还是客户端
- 浏览器名称或电脑操作系统
- 控制台 Console 页面的截图（按 F12 可打开控制台）

**Comment by rock-dengpaoer** [comment]
windows10，双屏显示器，界面不显示，但程序是可以运行的，能够播放，暂停，下一首

**Comment by qier222** [comment]
可能是因为YesPlayMusic记住的窗口位置超过了屏幕分辨率，导致整个窗口都在可视区域外

**Comment by rock-dengpaoer** [comment, marked as answer]
解决了，谢谢了，从新调整了一下显示器的排序
